### PR TITLE
Codebase stewardship

### PR DESCRIPTION
**Pull request by bvhme**
I've updated the codebase stewardship activity to reflect the latest thinking.

It has also become clear to me that it might be worthwhile to expand on the list of activities.

This is the diagram as we updated it in the last session:

Screenshot from 2019-08-07 13-47-13

### CODE REVIEW

**Comment by bvhme** [comment]
@ElenaFdR I've rewritten all of the items except for the criteria. I think those are best left for another time.

File: activities/codebase-stewardship/for-existing-projects.md [hunk 1]
```diff
@@ -4,18 +4,21 @@ type: Resource
 
 # Codebase stewardship for existing projects
 
-This explains:
-* what organizations that want a existing project to be in [codebase stewardship](index.md) have to do
-* what we do to support them in this process
+Codebase stewardship can be provided to a codebase after it has already been developed.
+In that case there might be some important considerations about how to transition and budget for that.
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/for-existing-projects.md]:
```suggestion
The developing organization and the Foundation for Public Code work together to transition an existing codebase, though each organization has their own responsibilities.
```

File: activities/codebase-stewardship/activities.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+type: Resource
+---
+
+# Activities performed in codebase stewardship
+
+Split into four domains; codebase stewardship is at least:
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/activities.md]:
```suggestion
Codebase stewardship activities take place in 4 domains: community, product marketing, quality and support.
```

File: activities/codebase-stewardship/activities.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+* Community stewardship
```
Comment (ElenaFdR) [inline activities/codebase-stewardship/activities.md]:
```suggestion
Community stewardship activities include:
```

**Pull request merged**